content/posts/android-on-fdroid.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Android app now on F-Droid'
+subtitle: 'Available to download right now!'
+author: 'Niels van Velzen'
+githubusername: 'nielsvanvelzen'
+date: 2021-03-01T00:00:00-00:00
+justify: 'center'
+---
+
+We're happy to announce the direct availability of the mobile Android app to the F-Droid store. The F-Droid version does not contain any proprietary libraries and is 100% reproducible.
+
+<!--more-->
+
+## Missing features
+
+Unfortunately, the library required to support casting is proprietary, meaning Google does not publish the source code. Because of this, we had to remove it for the F-Droid version. Fortunately, the remote control feature still works.
```

Comment:
```suggestion
Unfortunately, the library required for casting support is proprietary, meaning that it's not available under a free license and Google doesn't publish the source code. Because of that, we had to remove it in the F-Droid version. Fortunately, the remote control feature still works.
```